presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -9,287 +9,448 @@ Except for the limit on queued queries, when a resource group runs out of a reso
+     sub-group is computed based on the weights for all currently eligible sub-groups. The sub-group
+     with the least concurrency relative to its share is selected to start the next query.
 
-    * ``weighted``: queued queries are selected stochastically in proportion to their priority
-      (specified via the ``query_priority`` :doc:`session property </sql/set-session>`). Sub groups are selected
-      to start new queries in proportion to their ``schedulingWeight``.
+   * ``weighted``: queued queries are selected stochastically in proportion to their priority,
+     specified via the ``query_priority`` :doc:`session property </sql/set-session>`. Sub groups are selected
+     to start new queries in proportion to their ``schedulingWeight``.
 
-    * ``query_priority``: all sub-groups must also be configured with ``query_priority``.
-      Queued queries will be selected strictly according to their priority.
+   * ``query_priority``: all sub-groups must also be configured with ``query_priority``.
+     Queued queries are selected strictly according to their priority.
 
-* ``schedulingWeight`` (optional): weight of this sub-group. See above.
-  Defaults to ``1``.
+* ``schedulingWeight`` (optional): The weight for the resource group when the parent group uses the "weighted"
+  scheduling policy. A higher weight means that the group gets a larger share of the parent group's
+  resources.
 
-* ``jmxExport`` (optional): If true, group statistics are exported to JMX for monitoring.
+* ``jmxExport`` (optional):  If set to true, the statistics of the resource group will be exported via JMX.
   Defaults to ``false``.
 
-* ``perQueryLimits`` (optional): specifies max resources each query in a
-  resource group may consume before being killed. These limits are not inherited from parent groups.
-  May set three types of limits:
+* ``subGroups`` (optional): list of sub-groups.A list of sub-groups within the resource group.
+  Each sub-group can have its own set of properties.
 
-    * ``executionTimeLimit`` (optional): Specify an absolute value (i.e. ``1h``)
-       for the maximum time a query may take to execute.
+.. _scheduleweight-example:
 
-    * ``totalMemoryLimit`` (optional): Specify an absolute value (i.e. ``1GB``)
-       for the maximum distributed memory a query may consume.
+Scheduling Weight Example
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    * ``cpuTimeLimit`` (optional): Specify an absolute value (i.e. ``1h``)
-       for the maximum CPU time a query may use.
+Schedule weighting is a method of assigning a priority to a resource. Sub-groups
+with a higher scheduling weight are given higher priority. For example, to
+ensure timely execution of scheduled pipelines queries, weight them higher than
+adhoc queries.
+The scheduling weight in PrestoDB's Resource Groups is used when the parent group's
+scheduling policy is set to ``weighted``. It determines how resources are distributed
```

Comment:
Does "It determines how resources are distributed among the subgroups. " add to the meaning in this paragraph, or does it repeat what is already said? 

Consider if you can delete this sentence and the paragraph still means what you want it to say.